### ISSUE TITLE
Formula 1 Championship Edition SLUS-00546 Incorrect Display at bottom ~3% of screen in race

### ISSUE THREAD

**Issue by VirtuaCop** [MiSTer-devel/PSX_MiSTer#264]
http://redump.org/disc/15537/

Formula 1 Championship Edition SLUS-00546 

When in a race, the very bottom horizontal area of the display is displaying what appears to be elements from the top of the render area (trees, clouds, overhead advertisements, etc) This area updates dynamically as the game is rendered. This does not happen on original hardware. This is visible on both analog video output and HDMI video output.

This was reported in the discord by "DavesGameRoom" in May 2nd, 2022 but i don't see it on github. At that time the lower horizontal display area was a flickering color only. In the current build it seems to be displaying visuals that would normally appear at the top of the screen.

![Screenshot 2023-10-08 00-23-38](https://github.com/MiSTer-devel/PSX_MiSTer/assets/77247216/64e9decd-8b33-4dc8-9cf4-5eee162c1b46)
![Screenshot 2023-10-08 00-17-35](https://github.com/MiSTer-devel/PSX_MiSTer/assets/77247216/19801ed9-64ef-435f-a0de-ce63f874e94a)

**Comment by VirtuaCop** [comment]
I noticed that in the main menu options (not the in game options) there is a "Display area" type option. Moving this down will cause the incorrect display area to be hidden and more of the above area to be rendered correctly.
![Screenshot 2023-10-08 00-53-17](https://github.com/MiSTer-devel/PSX_MiSTer/assets/77247216/feb5c52b-fc84-43db-9d84-19c8e8b58e40)
![Screenshot 2023-10-08 00-53-52](https://github.com/MiSTer-devel/PSX_MiSTer/assets/77247216/c9e64f91-0917-4e0e-af77-ab9a3afe1738)

**Comment by VirtuaCop** [comment]
I noticed some strange behavior in this "screen setup" screen when moving the screen all the way down, the screen appeared to be magnified and on the HDMI display, the blue screen setup screen became larger than the 4:3 display area. 

On the CRT display nothing unusual happened, the blue area moved up and down.

https://www.youtube.com/watch?v=8pZ1XqlC2HU

**Comment by VirtuaCop** [comment]
Yes this would be considered a workaround. I just have been playing/sampling the whole library as they released in chronological order and this was one of October's games that I noticed an issue with and figured I'd report it. Thank you

**Comment by adamshell** [comment]
I can still reproduce this with core version 24.09.22, on both analog and HDMI. The screen adjustment workaround mentioned by @VirtuaCop still works fine.

Formula 1 '97 (SLES-00859) is the same game for PAL regions and does not have this issue.
